pkg/logger/logger.go

```diff
@@ -76,6 +76,11 @@ func (l *SugarLogger) With(args ...interface{}) *SugarLogger {
 	}
 }
 
+func (l *SugarLogger) IsDebug() bool {
+	return l.conf.Level.Level() == zap.DebugLevel
+}
```

Comment:
why no mutex.Lock()?

Reply:
Or maybe, after looking into the SugarLogger, why mutex at all? as the levels are atomic?
Plus, the underlying logger has `Enabled(level)`, which seems like the right thing to use.

Reply:
Changed to use the suggested API.